### ISSUE TITLE
IMPORTANT: MAINTAINER NEEDED ASAP!

### ISSUE THREAD

**Issue by adolfintel** [adolfintel/Windows10-Privacy#45]
Hello everyone,
I've been maintaining this guide since 2015 when the first version of Windows 10 came out. To be fair, it was already clear back then that Microsoft had decided to drive power users away from Windows, but I liked the OS so I persevered, and tried my best to make it as usable as possible using this guide.
With the release of the 1903 update a few weeks ago, which broke a lot of things on my computer, I just couldn't take it anymore; I'm sorry to let you down but Windows nowadays is as unstable as a crappy arch linux distro, I cannot be productive if the OS keeps breaking with every update.

So that's it, I have decided to quit Windows, and I'm now using Linux (Kubuntu, to be specific) as my main OS for everything except games (for which I'll keep a machine with Windows 10 LTSC, offline). As a result, 1903 will be the last update for this guide unless some good soul wants to take up the task of maintaining this project.

I know there are some skilled users out there who can do it, if you're interested, write me at info@fdossena.com or leave a comment under this issue.

**Comment by rugk** [comment]
Hehe, great. Maybe you should start your guide/Readme here with a big paragraph on "Consider switching to a different operation system"? :laughing: 

I'd suggest to link to these Linux beginner resources:
* https://computefreely.org/
* https://librehunt.org/
* education: https://linuxjourney.com/
* buy preloaded: https://linuxpreloaded.com/
* for gamers: https://www.gamingonlinux.com/ https://www.youtube.com/watch?v=Co6FePZoNgE

**Comment by playgithub** [comment]
I switched to Linux(Debian) / FreeBSD several times, but switched back to Windows at last, because Windows is much more easy to use. Is it worth to use Windows with some privacy leaking?

For example, if using chome, does M$ know what site I have visited?

**Comment by adolfintel** [comment]
@playgithub It depends on how much importance you place on your privacy and your data.
If you use Chrome, it shouldn't send anything to Microsoft, but it will send everything to Google, and that's even worse.

**Comment by playgithub** [comment]
Will Firefox send personal infos to Mozilla?
Maybe I should choose chromium, it doesn't send personal infos, does it?

**Comment by adolfintel** [comment]
Firefox sends personal information if you have telemetry enabled or if you log in with a firefox account.
Chromium is pretty much the same as regular chrome. Ungoogled-chromium and Iridium are modified to prevent it.

**Comment by rugk** [comment]
They all send some data, the thing is what they send and whether you can opt-out etc. And whether you can trust/verify what they send. Here Chrome is a proprietary browser, even though it's build on the [FLOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) browser Chromium. Firefox is fully open-source though (so you can verify what data it sends) and [they have a pretty nice and transparent privacy policy](https://www.mozilla.org/privacy/firefox/) and even feature [privacy as their first item on the website](https://www.mozilla.org/firefox/).

**Comment by rugk** [comment]
Anyway, this should not get into a browser discussion. This has nothing to do with this issue or even this repo.

**Comment by qwertychouskie** [comment]
> I switched to Linux(Debian) / FreeBSD several times, but switched back to Windows at last, because Windows is much more easy to use.

There's your problem, Debian and especially *BSD aren't exactly known for their user-friendliness.  Try Ubuntu, Xubuntu (my personal fave for day-to-day desktop work), or Mint, you'll get a much better experience.

**Comment by bmmcwhirt** [comment]
Just a quick note to say thank you for all you have done. I do hope you find a new maintainer as this is very useful. I mostly run Linux myself but at work I maintain the computers for a health company. Since come January we can no longer legally run Windows 7(HIPAA Requires an OS that has security updates) we will be upgrading and all this will come in very handy as we have limited bandwidth and with 200 computers trying to call home that adds up. 

Again, thank you for your service all these years.

**Comment by adolfintel** [comment]
@bmmcwhirt Windows 10 LTSC might be a good option for your company, it comes with no crap preinstalled and once you disable defender and use a firewall it's really not that bad.

**Comment by bmmcwhirt** [comment]
> @bmmcwhirt Windows 10 LTSC might be a good option for your company, it comes with no crap preinstalled and once you disable defender and use a firewall it's really not that bad.

Thanks I will look into that. We normally just get bulk licenses from TechSoup since we are a non-for-profit hospital. I'll see if they offer that at all.

**Comment by kophe** [comment]
I've been following your efforts for some time now, @adolfintel, and while I'm not as skilled as you, I built an automated script implementing your guide with some other patches, like Win10-Initial-Setup-Script , tron, O&OSU10 and so on.

I keep hopping between Fedora and Solus on my personal notebook, but like @bmmcwhirt, I have to use Windows at my workplace, and LTSC has been our choice since 2016 (LTSB). With this in mind,, I feel like I may contribute with a thing or two on the next updates. Count me in with anything I can help.

**Comment by adolfintel** [comment]
@kophe That's great, write me at info@fdossena.com so I have your email address.

**Comment by bmmcwhirt** [comment]
@kophe I have been working on an interactive script myself using what I've learned from this project. If you would like to compare notes I'd be happy to show what I have. It's not clean or elegant yet by any means.

**Comment by adolfintel** [comment]
@bmmcwhirt Sure, I'd like to see that script

**Comment by kophe** [comment]
I use @adolfintel's guide with a ton of custom implementations, mostly from [Tr0n](https://github.com/bmrf/tron), [Win10-Initial-Setup-Script](https://github.com/Disassembler0/Win10-Initial-Setup-Script) and random guides I've been studying to debloat, tweak and customize Windows 10.

Here's my "automated" version of his code with few adjustments: [https://github.com/kophe/win10_debloat](https://github.com/kophe/win10_debloat)

Basically, you run Stage_1.bat with admin permissions and follow the instructions on the prompt.

I'm no developer, just a hobbyist trying to contribute, but I think I still need to convert all the code to PowerShell and merge the duplicated code from all the guides I'm using before putting it online. This might take a while... :v

**Comment by Rehorek** [comment]
@adolfintel Thank you for sharing your project. For me it's going to be pretty useful. I'm dealing with it, so I'd like to ask you. Is there a way to do all your steps having two acounts: Aministrator and Standard user. I used to work on a Standard user account and it worked for me pretty well. It was on Windows 7.
After installing system I've created a Standard user. Did your scenario on an Administrator account and got back to Standard one and many of the changes have been rolled back. I saw your comment that creating new users won't work. However, I thought that maybe once the user is already created before doing your script it wouldn't  affect it.

**Comment by adolfintel** [comment]
@Rehorek I haven't tested it, but you should be able to create the 2 accounts, switch between the two, and then run the guide on both. On the standard account, use the credentials of the admin account to run commands as administrator.

**Comment by bmmcwhirt** [comment]
@Rehorek There are a few things to look at here. 

AppxPackage are different than WindowsPackage in that AppxPackage are per user and WindowsPackage are system wide. That means if you remove a WindowsPackage it's gone from everyone. With a AppxPackage you can remove it from the current user as such:

Get-AppxPackage -AllUsers {packageName} | Remove-AppxPackage  [This only removes it for the current user in my personal experience]

Get-AppxPackage -AllUsers {packageName} | Remove-AppxPackage -AllUsers [This will remove the package from all existing users]

However that is only part of the battle. AppxPackage can also be provisioned. That means that if you add a new user to the system, when they first login any provisioned apps will be installed for that user. This is especially true in a Domain controlled environment where multiple users may sign into any given machine. So there is one more step you have to take.

Get-AppXProvisionedPackage -Online {packageName} | Remove-AppxProvisionedPackage -Online

That will remove an app from provisioning so if a new user signs in it will not be installed.

**Comment by Rehorek** [comment]
@adolfintel Thank you for your response. I did that and it's a bit mess. Some applications were removed and some not. I've used get-appxpackage *app name* | remove-appxpackage for the rest which left. I'm not an experienced user, but in Windows 7 everything seemed to be simpler with rights and interdependence. 

@bmmcwhirt  Thanks for your explanation I did some changes in @adolfintel's, but I didn't see much difference. 

Anyway thank you. I don't want to bother you as I need to learn more about Windows 10 administration. I guess it might clear out some issues for me :-)
